### PR TITLE
Adjust call icon positioning in participant list

### PR DESCRIPTION
**Pull request by PVince81**
Fixed call icon vertical alignment.
Adjusted ellipsis width calculation to take the icon into account when
present.

@nickvergessen as discussed

### Before:
image

### After:
image

with status line:
image

### CODE REVIEW

**Comment by PVince81** [comment]
optional bonus tooltip here: https://github.com/nextcloud/spreed/pull/4621

**Comment by nickvergessen** [comment]
The icon seems out of position for non-moderators:
Bildschirmfoto von 2020-11-17 19-10-37

Also while inline, it looks weird to have the offset on your own call icon:
Bildschirmfoto von 2020-11-17 19-10-44

**Review by ma12-co — COMMENTED** [review]
flex on the container and then align-items: center?

**Comment by PVince81** [comment]
new state:
image

**Comment by PVince81** [comment]
@nickvergessen please re-review

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue [hunk 1]
```diff
@@ -360,23 +366,41 @@ export default {
...
+		width: calc(100% - 100px);
 		display: flex;
 		flex-direction: column;
+
+		&.has-call-icon {
+			/** reduce text width to have some distance from call icon */
+			padding-right: 5px;
+		}
+
+		&.has-call-icon {
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue:391]: ERROR in 
src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue
 391:3  ✖  Unexpected duplicate selector ".participant-row__user-wrapper.has-call-icon", first used at line 386   no-duplicate-selectors

**Review by nickvergessen — APPROVED** [review]
Works

**Pull request merged**